api/upload_route.py

```diff
@@ -0,0 +1,82 @@
+import uuid
+import os
+import requests
+
+from flask import Blueprint, request, jsonify
+from datetime import datetime
+
+from .cuckoo_service import upload_to_cuckoo, start_cuckoo_monitor
+from .log import create_log, update_log_stage, add_error_message
+
+
+UPLOAD_FOLDER = 'uploads'
+ALLOWED_MIME_MAGIC = {b'MZ'}
+ALLOWED_EXTENSIONS = {'exe', 'dll'}
+
+upload_bp = Blueprint('upload', __name__)
+
+os.makedirs(UPLOAD_FOLDER, exist_ok=True)
+
+@upload_bp.route('/upload', methods=['POST'])
+def upload_file():
+    if 'file' not in request.files:
+        return jsonify({'error': 'No file part.'}), 400

```

Comment:
建議用 `flask` 內的 `make_response`。

```suggestion
        return make_response({'error': 'No file part.'}, HTTPStatus.BAD_REQUEST)
```

記得要 
```python
from flask import make_respnose
from http import HTTPStatus
```